Documentation/EtcdLease.md

```diff
@@ -0,0 +1,38 @@
+# Overview
+The EtcdLease implement interface `grant` and `revoke` to new a lease and revoke a lease to etcd.
+It can manage the leases to send keep alive request for leases.
+
+
+## `startKeepAliveService` function
```

Comment:
## StartKeepAliveServe function

Remove the annotation? It looks strange. Try to click view to see the generated page.